test/overall_test.py

```diff
@@ -90,6 +90,17 @@
 >>> result = g.save_data(file_adr="")
 >>> result["status"]
 False
+>>> def f1(x, y):
+...    return math.cos(x**2*y) ** 1.926 - math.floor(x-y) ** 1.861 - math.floor(y**2*x)**1.688
+
+>>> def f2(x, y):
+...    return x - y**1.617 - math.ceil(y) ** 1.477 - abs(x**2 * y) ** 1.647 - math.cos(x*y)**1.668
+>>> g = GenerativeImage(f1, f2)
+>>> g.generate(seed=755398)
```

Comment:
Please add some tests here, to check that `data1` and `data2` does not include any complex numbers